robottelo/ui/subnet.py

```diff
@@ -60,19 +70,13 @@ def delete(self, subnet_name, really=True):
         self.delete_entity(
             subnet_name,
             really,
-            locators['subnet.display_name'],
             locators['subnet.delete'],
         )
 
-    def search_subnet(self, subnet_name):
+    def search_subnet_and_validate(self, subnet_name):
```

Comment:
Pretty long name. As it's already a subnet module, maybe rename it to `search_and_validate`? Not a blocker